### PR TITLE
Add label to android hello_world example

### PR DESCRIPTION
**Pull request by conradgrobler**

### CODE REVIEW

File: examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java [hunk 1]
```diff
@@ -131,11 +135,23 @@ public void run() {
...
           .useTransportSecurity()
           .sslSocketFactory(getSocketFactory())
           .build();
     }
 
+    /** Gets the metadata associated with the empty label (public/untrusted). */
+    private Metadata getPublicUntrustedLabelMetadata() {
+      // TODO(#1066): Use a more restrictive Label.
+      Metadata result = new Metadata();
+      Label label = Label.newBuilder().build();
```
Comment (ipetr0v) [inline examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java:148]: Is it a public label?

Comment (conradgrobler) [reply examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java:148]: Yes, a label with no secrecy tags is public.

File: examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java [hunk 2]
```diff
@@ -41,6 +44,7 @@
 
 /** Main activity for the Oak Android "Hello, World" app. */
 public class MainActivity extends Activity {
+  private static final String OAK_LABEL_GRPC_METADATA_KEY = "x-oak-label-bin";
```
Comment (tiziano88) [inline examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java:47]: Maybe add a comment with a link to where this is defined in the Rust Runtime impl.

Comment (conradgrobler) [reply examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java:48]: Done.

**Pull request merged**